pages/common/git-cherry.md

```diff
@@ -0,0 +1,17 @@
+# git cherry
+
+> Find commits yet to be applied to upstream.
+> Commits prefixed with - have an equivalent; those with + do not.
```

Comment:
Perhaps this will read better?
```suggestion
> Find commits that have yet to be applied upstream.
```

Reply:
Regarding line three, I just took the blurb from git's docs, but I'd agree that "that have" adds clarity.

As for line four, I was hoping to provide a little context because it's frankly confusing, but I suppose that's just once there's any level of complexity, so it'd be fine to leave out.